.gitignore

```diff
@@ -0,0 +1,34 @@
+# Ignore Maven Compiled Project Files
+*/target/
+
+# Ignore IntelliJ Project Information
+*.iml
+.idea/
+
+# Ignore Sublime Text Project Files
+*.sublime-project
+*.sublime-workspace
+sftp-config.json
+
+# Ignore Eclipse Project Information
+*.settings
+*.project
+*.classpath
+dependency-reduced-pom.xml
+
+# Ignore Excel Files
+*.xlsx
+
+# Ignore local log files
+*.log
+*.remote-sync.json
+
+# Ignore Generated HTML README Files
+README.html
+
+# Ignored installed NPM modules
+node_modules/
```

Comment:
Since this action will use the dependencies to run the JavaScript Action, we cannot have node_modules in the .gitignore here

Reply:
@penwoodjon  haha, good catch.  Removed that along with the `test-runner.js` which I don't think is really needed for this repo.